torch/csrc/jit/passes/constant_propagation.cpp

```diff
@@ -336,6 +338,13 @@ struct ConstantPropagator {
   }
 
   void ConstantPropagation(Node* n) {
+    auto allowed = JIT_BISECT();
```

Comment:
Nit: a random thought, I've just realized that while this API will probably mostly be used for the bisection, the macro itself doesn't do any bisection: it simply enables/disables the optimization. Maybe we should pick a different name that better reflects what it does?